app/src/main/res/xml/preferences_import_export.xml

```diff
@@ -27,6 +27,13 @@
                 android:summary="@string/opml_import_summary"/>
     </PreferenceCategory>
 
+    <PreferenceCategory android:title="@string/favorites">
+        <Preference
+            android:key="prefFavoritesExport"
+            android:title="@string/favorites_export_label"
+            android:summary="@string/favorites_export_summary"/>
```

Comment:
Please use the same indentation as the rest of the file. Wouldn't it make sense to display the item in the HTML category? Otherwise, we have two categories with only one item each.